tests/views/test_respondents.py

```diff
@@ -201,3 +108,74 @@ def test_change_respondent_status(self, mock_request):
                                     f"?tab=respondents&change_flag=ACTIVE",
                                     follow_redirects=True)
         self.assertEqual(response.status_code, 200)
+
```

Comment:
This is the UI style testing that I've added, it negates the need to mock large amounts of other functions, and allows you to run UI style tests, without including them in the acceptance test repo (acceptance tests and UI tests are different).

Included here as both testing for this, and also a useful talking point as to how we want to proceed - if we decide on a different direction, I'll swap it for something else.